.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: r
+cache: packages
+before_install:
+  - cd dataCompareR
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
This avoids problems with any tests that expect a screen / GUI
see [here](https://docs.travis-ci.com/user/gui-and-headless-browsers/)